### DISCUSSION TITLE
How to nsert Empty page after index(TOC) page in PDF render

### DISCUSSION THREAD

**Discussion by bharathgs** [quarto-dev/quarto-cli#2777] [Q&A] [answered]
I was wondering if there's a way to insert an empty/blank page after the index/TOC page while rendering a document as a pdf. Currently, if you generate a quarto doc with an index/toc page - the actual content begins right after the TOC page.

**Comment by melissavanbussel** [comment, marked as answer]
You can add a `\pagebreak` after your YAML, before the main part of your document begins. For example: 

~~~
---
title: "Your title here"
format: pdf
editor: visual
toc: true
---

\pagebreak

# Heading 1

Some text. 

## Heading 2

Some more text.

## Another heading 2

Even more text.
~~~

This produces the following: 

<img width="204" alt="image" src="https://user-images.githubusercontent.com/28693927/194977353-8699571a-0566-4167-a108-37378b58a746.png">
